src/Menus/HasMenuIcons.php

```diff
@@ -7,45 +7,68 @@
  *
  * For the full copyright and license information, please view
  * the LICENSE file that was distributed with this source code.
+ *
+ * php version 8.0
+ *
+ * @category Menus
+ * @package  Bonfire
+ * @author   Lonnie Ezell <lonnieje@gmail.com>
+ * @license  MIT https://opensource.org/licenses/MIT
+ * @link     https://github.com/lonnieezell/Bonfire2/
+ * @see      https://github.com/lonnieezell/Bonfire2/
  */
 
 namespace Bonfire\Menus;
 
+/**
```

Comment:
Why do we need duplicates in the description?

Reply:
I removed duplicate File DocBlock in last commit.